framework/src/modules/chain/submodules/process_transactions.js

```diff
@@ -137,24 +138,72 @@ class ProcessTransactions {
 
 		await Promise.all(transactions.map(t => t.prepare(stateStore)));
 
-		const transactionsResponses = transactions.map(transaction => {
-			const transactionResponse = transaction.apply(stateStore);
-			roundInformation.apply(stateStore, transaction);
-			stateStore.transaction.add(transaction);
-			return transactionResponse;
+		return {
+			transactionsResponses: transactions.map(transaction => {
+				const transactionResponse = transaction.apply(stateStore);
+
+				roundInformation.apply(stateStore, transaction);
+				stateStore.transaction.add(transaction);
+
+				transactionResponse.status = TransactionStatus.OK;
```

Comment:
Maybe add a comment here which is something like this: 
```suggestion
                                 // We are overriding the status of transaction because it's from genesis block 
				transactionResponse.status = TransactionStatus.OK;
```